test/performance/odp_l2fwd.c

```diff
@@ -1247,6 +1517,14 @@ static void usage(char *progname)
 	       "                          1: Create a pool per interface\n"
 	       "  -n, --num_pkt <num>     Number of packets per pool. Default is 16k or\n"
 	       "                          the maximum capability. Use 0 for the default.\n"
+	       "  -u, --vector_mode       Enable vectors mode.\n"
```

Comment:
It would be good to clarify here that vector mode is supported only with scheduler pktion modes (1-3).

Reply:
Will add this note.